### PR TITLE
sdl_image: add version 2.6.3

### PR DESCRIPTION
**Pull request by toge**
Specify library name and version:  **sdl_image/***

Retry https://github.com/conan-io/conan-center-index/pull/19956.

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/sdl_image/all/conanfile.py [hunk 1]
```diff
@@ -47,23 +52,35 @@ class SDLImageConan(ConanFile):
...
+            copy(self, "CMakeLists.txt", src=self.recipe_folder, dst=self.export_sources_folder)
 
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
         if not is_apple_os(self):
             del self.options.imageio
+        if Version(self.version) < "2.6":
+            if self.settings.os != "Windows":
+                del self.options.wic
```
Comment (RubenRBS) [inline recipes/sdl_image/all/conanfile.py:80]: Just to confirm, wic has been an option all this time but we were just missing it right?

**Review by RubenRBS — COMMENTED** [review]
Thanks a lot! The recipe looks great, I'll approve once I can check locally that the new options build when set to True :)

**Comment by RubenRBS** [comment]
@toge I've directly pushed a commit thinking I was in my own branch, sorry about that, I can revert if needed and you want to fix it yourself. There was a missing requires when using one of the new versions

**Comment by toge** [comment]
@RubenRBS 
Thanks a lot!
You're welcome!

**Review by RubenRBS — APPROVED** [review]

**Review by jwillikers — APPROVED** [review]

**Pull request merged**